enterprise/cmd/frontend/internal/repos/webhooks/bitbucket_server_handler.go

```diff
@@ -39,9 +39,16 @@ func (g *BitbucketServerHandler) handlePushEvent(ctx context.Context, payload an
 		return errors.Newf("expected BitbucketServer.PushEvent, got %T", payload)
 	}
 
-	repoName, err := bitbucketServerNameFromEvent(event)
+	cloneURL, err := bitbucketServerCloneURLFromEvent(event)
 	if err != nil {
-		return errors.Wrap(err, "handlePushEvent: get name failed")
+		return errors.Wrap(err, "getting clone URL from event")
```

Comment:
Maybe the suggestion below? I'm thinking of this while reading it inside in a log / UI and the explicit note about the "failed to" part might be helpful? Feel free to ignore if you think that's not critical and already obvious.

(Also for the similar error message right below on line 48, but I'm skipping the comment / suggestion)

```suggestion
		return errors.Wrap(err, "failed to get clone URL from event")
```

Reply:
I think since it's already in an error we don't need to mention that it failed